statsmodels/regression/tests/test_special_linear.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jan 18 23:39:45 2016
+
+Author: Josef Perktold
+License: BSD-3
+"""
+
+import numpy as np
+from scipy import sparse
+import pandas as pd
+
+from statsmodels.regression.linear_model import OLS
+from statsmodels.regression.special_linear_model import (
+      OLSAbsorb, _group_demean_iterative, cat2dummy_sparse)
+from statsmodels.tools._sparse import PartialingSparse, dummy_sparse
+
+from numpy.testing import assert_allclose
+
+# COMPAT: for numpy <= 1.7 until we increase minimum
+from statsmodels.compat.scipy import NumpyVersion
+np_smaller_108 = True if (NumpyVersion(np.__version__) < '1.8.0') else False
+from numpy.testing import dec
+from nose import SkipTest
+
+
+class CompareModels(object):
+
+    def test_attributes(self):
+        res1 = self.res1
+        res2 = self.res2
+        slice_res1 = self.slice_res1
+        slice_res2 = self.slice_res2
+
+        good = ['HC0_se', 'HC1_se', '_data_attr', 'aic', 'bic', 'bse',
+                'centered_tss', 'df_model', 'df_resid', 'ess', 'f_pvalue',
+                'fvalue', 'k_constant', 'llf', 'mse_model', 'mse_resid',
+                'mse_total', 'nobs', 'params', 'pvalues', 'resid_pearson',
+                'rsquared', 'rsquared_adj', 'scale', 'ssr', 'tvalues',
+                'use_t', 'wresid']
+
+        known_fails = ['HC2_se', 'HC3_se', '_wexog_singular_values',
+                       'condition_number', 'eigenvals', 'fittedvalues',
+                       'resid', 'uncentered_tss']
```

Comment:
we need doc comment what predict does, and what fittedvalues and resid are. They don't take fixed effects into account ?